src/SprykerShop/Yves/CustomerPage/CustomerPageConfig.php

```diff
@@ -37,7 +37,7 @@ class CustomerPageConfig extends AbstractBundleConfig
     /**
      * @uses \Spryker\Zed\Customer\CustomerConfig::MAX_LENGTH_CUSTOMER_PASSWORD
      */
-    protected const MAX_LENGTH_CUSTOMER_PASSWORD = 72;
+    protected const MAX_LENGTH_CUSTOMER_PASSWORD = 18;
```

Comment:
this is the max length considering 72 (max bcrypt byte size) / 4 (max UTF-8 byte size)